core/src/main/java/com/netflix/conductor/core/execution/mapper/ForkJoinDynamicTaskMapper.java

```diff
@@ -277,42 +263,43 @@ TaskModel createDynamicForkTask(
      * this#getMappedTasks(TaskMapperContext)} at the end to add a join task to be scheduled after
      * all the fork tasks
      *
-     * @param workflowInstance: A instance of the {@link WorkflowModel} which represents the
-     *     workflow being executed.
+     * @param workflowModel: A instance of the {@link WorkflowModel} which represents the workflow
+     *     being executed.
      * @param joinWorkflowTask: A instance of {@link WorkflowTask} which is of type {@link
      *     TaskType#JOIN}
      * @param joinInput: The input which is set in the {@link TaskModel#setInputData(Map)}
      * @return a new instance of {@link TaskModel} representing a {@link TaskType#JOIN}
      */
     @VisibleForTesting
     TaskModel createJoinTask(
-            WorkflowModel workflowInstance,
+            WorkflowModel workflowModel,
             WorkflowTask joinWorkflowTask,
             HashMap<String, Object> joinInput) {
         TaskModel joinTask = new TaskModel();
```

Comment:
Should this be replaced with `createTaskModel` as well?

Reply:
This can't be, because the `joinWorkflowTask` is not part of the `TaskMapperContext`. See https://github.com/Netflix/conductor/blob/main/core/src/main/java/com/netflix/conductor/core/execution/mapper/ForkJoinDynamicTaskMapper.java#L212-L225